components/rmrk/Gallery/CollectionItem.vue

```diff
@@ -172,6 +182,10 @@ export default class CollectionItem extends mixins(
   protected priceData: [ChartData[], ChartData[]] | [] = []
   private statsLoaded = false
   private queryLoading = 0
+  public eventsOfNftCollection: Interaction[] | [] = []
+  public selectedEvent = 'all'
+  public priceChartData: [Date, number][][] = []
```

Comment:
So this is 2D Array of tuples? `<Array<Array<[Date, number]>>`

Reply:
Yes it is, like the GalleryItem.vue. It's needed in the History.vue by default on the created(), after finish to process the Interaction[]. Maybe some change is needed on the History component after?